### PR TITLE
feat(lib/Utils/Log.php): Added DocBlocks

### PR DESCRIPTION
**Pull request by vandebled**

### CODE REVIEW

File: lib/Utils/Log.php [hunk 1]
```diff
@@ -30,6 +74,18 @@ public static function consoleDebug($data, $postpone = true, $title = 'PHP', $lo
...
     }
 
+    /**
+    * Pretty prints a variable.
+    *
+    * Prints human-readable information about a variable.
+    * Print will be wrapped between <pre> tags.
+    * Will also output the data type, the filename and line number.
+    *
+    * @since 0.2.0
```
Comment (domtra) [inline lib/Utils/Log.php]: what do we do with something like this? `0.2.0` is not even released yet, so this number could be anything

File: lib/Utils/Log.php [hunk 2]
```diff
@@ -43,6 +99,14 @@ public static function printDebug($data, $postpone = true)
...
 
+    /**
+    * Echoes a variable.
+    *
+    * @since 0.2.0
+    *
+    * @param mixed    $data     Data to be printed.
+    * @param boolean  $postpone Postpone printing to the wp_footer action.
+    */
     protected static function echoDebug($data, $postpone)
```
Comment (domtra) [inline lib/Utils/Log.php]: should we actually add docblocks for protected methods?

File: lib/Utils/Log.php (hunk 1 above)
Comment (vandebled) [inline lib/Utils/Log.php]: yes, what should we do?

File: lib/Utils/Log.php [hunk 3]
```diff
@@ -54,6 +118,13 @@ protected static function echoDebug($data, $postpone)
         }
     }
 
+    /**
+    * Returns a filename and line number by performing a debug_backtrace.
+    *
+    * @since 0.1.0
+    *
+    * @param number $depth  depth to return.
```
Comment (Qakulukiam) [inline lib/Utils/Log.php]: not sure of this makes a difference, but should it be `number` / `int` / `integer`?

File: lib/Utils/Log.php (hunk 2 above)
Comment (vandebled) [inline lib/Utils/Log.php]: They are in WP. What do you think?
@Qakulukiam ?

Comment (Qakulukiam) [inline lib/Utils/Log.php]: Doesn't hurt? But maybe not make it mandatory for now?

Comment (vandebled) [inline lib/Utils/Log.php]: ok maybe i don't for the next files then

File: lib/Utils/Log.php [hunk 4]
```diff
@@ -4,21 +4,65 @@
...
+    */
     public static function console($data, $postpone = true)
     {
         self::consoleDebug($data, $postpone);
     }
 
+    /**
+    * Outputs an error message to the Web Console.
+    *
+    * Outputs a JavaScript `error.log`.
```
Comment (Qakulukiam) [inline lib/Utils/Log.php]: this is actually `console.error` right?

**Pull request merged**